doc/Uplink.xml

```diff
@@ -292,20 +292,20 @@
       <para>Note that for the following discussion the roles of client and server are swapped.</para>
       <para>The remote client shall authenticate itself using a valid server certificate. The local
         service shall verify the validity of the remote certificate according to RFC 6125.</para>
+      <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
       <section>
         <title>mTLS authentication</title>
+        <para>To signal that a device supports this authentication mechanism, it shall include <literal>mTLS</literal> in the supported <literal>AuthorizationModes</literal> in the GetServiceCapabilities response.</para>
```

Comment:
Paraphrase proposal to use consistent language while referring to capabilities as requirements from device perspective.

>Device signaling support for mTLS based authentication shall include mTLS in the list of AuthorizationModes of GetServiceCapabilities Response.

Reply:
Applied